examples/udt/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 #![no_std]
-use stellar_contract_sdk::{contractimpl, contracttype, ConversionError, IntoEnvVal};
+use stellar_contract_sdk::{contract, contractimpl, contracttype, ConversionError, IntoEnvVal};
+
+contract!();
```

Comment:
@jonjove We have a lot of options for how we evolve this but there's going to be a macro that we should find a way to require gets included. The macro will embed some metadata into contract. Probably we can embed other test related registrations, vtable generation, etc, in here possibly.